### PR TITLE
chore: migrated indicative cell to tsx

### PR DESCRIPTION
**Pull request by aizad-deriv**
## Changes:

Please include a summary of the change and which issue is fixed below:
-   ...

## When you need to add unit test

-   [ ] If this change disrupt current flow
-   [ ] If this change is adding new flow

## When you need to add integration test

-   [ ] If components from external libraries are being used to define the flow, e.g. @deriv/components
-   [ ] If it relies on a very specific set of props with no default behavior for the current component.

## Test coverage checklist (for reviewer)

-   [ ] Ensure utility / function has a test case 
-   [ ] Ensure all the tests are passing

## Type of change

-   [ ] Bug fix
-   [ ] New feature
-   [ ] Update feature
-   [ ] Refactor code
-   [ ] Translation to code
-   [ ] Translation to crowdin
-   [ ] Script configuration
-   [ ] Improve performance
-   [ ] Style only
-   [ ] Dependency update
-   [ ] Documentation update
-   [ ] Release

### CODE REVIEW

File: packages/reports/src/Components/indicative-cell.tsx [hunk 1]
```diff
@@ -1,11 +1,28 @@
...
 
-const IndicativeCell = ({ amount, currency, contract_info, is_footer, onClickSell, is_sell_requested }) => {
-    const [movement, setMovement] = React.useState(null);
+type TIndicativeCell = {
+    amount: number;
+    contract_info: object;
+    currency: string;
+    status: string;
+    is_footer: boolean;
+    is_sell_requested: () => void;
```
Comment (sergei-deriv) [inline packages/reports/src/Components/indicative-cell.tsx]: I have doubts about 'is_sell_requested'. In my opinion it's boolean or function which returns boolean: "() => boolean"

Comment (akmal-deriv) [reply packages/reports/src/Components/indicative-cell.tsx]: I don't think it's a function, should be just `boolean`.

Comment (akmal-deriv) [reply packages/reports/src/Components/indicative-cell.tsx]: Normally, we use camelCase for functions and snake_case for normal variables

Comment (aizad-deriv) [reply packages/reports/src/Components/indicative-cell.tsx]: Will take a look at this thank you for pointing it out 🙏🏼

**Review by sergei-deriv — APPROVED** [review]

**Review by akmal-deriv — APPROVED** [review]

File: packages/reports/src/Components/indicative-cell.tsx [hunk 2]
```diff
@@ -1,11 +1,28 @@
...
 import { Icon, Money, DesktopWrapper, ContractCard } from '@deriv/components';
 import { getCardLabels } from '_common/contract';
 import { connect } from 'Stores/connect';
+import { TRootStore } from 'Stores/index';
 
-const IndicativeCell = ({ amount, currency, contract_info, is_footer, onClickSell, is_sell_requested }) => {
-    const [movement, setMovement] = React.useState(null);
+type TIndicativeCell = {
+    amount: number;
+    contract_info: object;
```
Comment (mahdiyeh-deriv) [inline packages/reports/src/Components/indicative-cell.tsx]: please expand this object type due to the `contract_info` object

**Review by farabi-deriv — APPROVED** [review]

**Review by mahdiyeh-deriv — APPROVED** [review]

**Pull request merged**